setup.py

```diff
@@ -342,7 +342,7 @@ def main():
     }
     skw["cmdclass"]["develop"] = xdevelop
     skw["extras_require"] = {
-        "ptk": ["prompt-toolkit>=3.0"],
+        "ptk": ["prompt-toolkit>=3.0", "pyperclip"],
```

Comment:
does it make sense to add pyperclip to full too?

Reply:
since ptk is part of full, it will automatically get installed when full is used.

Reply:
Oh right!